plugins-compat-tester/src/test/java/org/jenkins/tools/test/PluginCompatTesterTest.java

```diff
@@ -142,6 +139,7 @@ public void testWithUrl() throws Throwable {
         }
     }
     
+    @Ignore("TODO broken by https://github.com/jenkinsci/active-directory-plugin/releases/tag/active-directory-2.17; figure out how to pin a version")
```

Comment:
Use of Incrementals in https://github.com/jenkinsci/active-directory-plugin/pull/102 seems to have broken these two tests. Anyway no tests here ought to be drawing plugin versions from the update center. (Perhaps we should simply remove this mode and only support `-war`? I do not know of anyone who actually uses it.)